tools/sggolangcilint/golangci.yml

```diff
@@ -60,10 +71,6 @@ linters:
     # [fast: true, auto-fix: true]
     - godot
 
-    # Check whether code was gofumpt-ed.
-    # [fast: true, auto-fix: true]
-    - gofumpt
```

Comment:
why is this removed in the new config?

Reply:
It was moved out of the `linters` section and is now a formatter, like with `gci`.